docs/enrollment_and_payment.md

```diff
@@ -0,0 +1,143 @@
+subsequent operations are performed with is contained in the asynchronous callback.
+
+```mermaid
+sequenceDiagram
+    participant Integrator
+    participant ePaymentPlatform
+    
+    Integrator->>ePaymentPlatform: RequestPayment
+    activate ePaymentPlatform
+    ePaymentPlatform-->>Integrator: 200 OK.
+    deactivate ePaymentPlatform
+    
+    ePaymentPlatform ->> ePaymentPlatform : Resolve Payment Status
+
+    ePaymentPlatform->>Integrator: Asynchronous Payment result callback!
+    activate Integrator
+    Integrator-->>ePaymentPlatform: 200 OK.
+    deactivate Integrator
+    note left of Integrator: The callback contains a paymentId <br/> which subsequent Transaction Operations are <br/> performed with.
+
+    alt Subsequent operations (Capture, Refund, Cancel)
+    
+    Integrator ->> ePaymentPlatform: Transaction Operations.
+    activate ePaymentPlatform
+    ePaymentPlatform-->>Integrator: 200 OK
+    deactivate ePaymentPlatform
+
+    ePaymentPlatform ->> ePaymentPlatform : Resolve Payment Operation
+
+    ePaymentPlatform->>Integrator: Asynchronous Payment Operation result callback!
+    activate Integrator
+    Integrator-->>ePaymentPlatform: 200 OK.
+    deactivate Integrator
+    
+    end 
+```
+
+### Creating a payment guidelines.
+
+``messageId``: The ``messageId``field is considered the Integrator's unique identifier of a PaymentRequest, and can be used in the case of a [Rollback Request](/assets/swagger/swagger_integrator_token_merchant_bankaxept/).
+of an ongoing payment. The ePaymentPlatform performs duplicate controls on the ``messageId`` field, and acts idempotent on requests with the same ``messageId``. Therefore, it *must* be unique pr separate payment request. Meaning that if multiple are done for the same Order (for example a retry due to a previously failed payment request.), a new ``messageId`` must be used.
+
+``merchantOrderReference``: The ``merchantOrderReference``field is considered a reference to the Merchant's Order which might be distinct from the Integrator's own ``messageId``.
+
+### Callbacks
+
+As an integrator processing payments you must integrate the [Payment Callback API](/assets/swagger/swagger_integrator_merchant_partner_bankaxept/) to receive asynchronous updates on the status of your payments.
+
+## Settlement and Cutoff
+To create a Settlement you must send a [Cutoff request](/assets/swagger/swagger_integrator_merchant_bankaxept/). 
+The Cutoff request must contain a `merchantId` and `batchNumber` which will trigger a settlement for all transactions tied to the corresponding `batchNumber`.
+The `batchNumber` is tied to a payment request and is returned in the asynchronous callback to the Integrator's Callback Server once a payment is successfully processed.
+In the case of an error or timeout (for example due to a network issue), the request may be retried.
+
+Once a Cutoff request is received the `batchNumber` will increase for all subsequent payment requests.
```

Comment:
Maybe some rewording, sounds a bit like the batchNumber will increase for each single payment request. More like the batchNumber will increase and the new value will be used for all subsequent payments until next cutoff, when the batchNumber will increase again.